tests/clients/test_mysqlclient.py

```diff
@@ -217,3 +217,93 @@ def test_error_capture(self):
         self.assertEqual(db_span.data["mysql"]["stmt"], 'SELECT * from blah')
         self.assertEqual(db_span.data["mysql"]["host"], testenv['mysql_host'])
         self.assertEqual(db_span.data["mysql"]["port"], testenv['mysql_port'])
+
+    def test_connect_ctx_mngr(self):
+        result = None
```

Comment:
The context manager does not create a new scope, the `result = cursor.execute(` statement is enough. Please remove this line, luckily this is not Pascal or ANSI C, where you have to declare the variables way ahead of assigning them.